Feat/rebrand

KillerX opened the pull request:
Rebranding images:

Known issues:

* Android Splash

andreasgan reviewed (changes requested):
for the ios splash, it looks like all 3 images are the same.
they also include a background. can you just use the logo alone and use a background on the .storyboard instead?
Storyboards have a visual editor in xcode btw.

KillerX commented:
@andreasgan I removed the splash images in iOs, and added the proper 2 layer icon to Andorid, so various shapes look ok now:

![image](https://user-images.githubusercontent.com/398737/208681190-37153b59-6b8f-4879-8593-81837450bd15.png)

andreasgan commented on ios/Runner/Info-Debug-prod.plist:
```diff
@@ -2,8 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>ITSAppUsesNonExemptEncryption</key>
```
curious why you removed this

KillerX replied:
Xcode just moved it down below to line 50. Didn't manually edit this file
